docs/source/getting_started/manufacturing/01_intro.ipynb

```
@@ -64,6 +64,34 @@
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import torch\n",
+    "\n",
+    "def get_device():\n",
+    "    \"\"\"Get the appropriate device for model inference.\"\"\"\n",
+    "    if torch.cuda.is_available():\n",
+    "        return \"cuda\"\n",
+    "    elif hasattr(torch.backends, \"mps\") and torch.backends.mps.is_available():\n",
+    "        return \"mps\"\n",
+    "    return \"cpu\"\n",
```

Ruff (0.14.0)

Shebang should contain `python`, `pytest`, or `uv run`

(EXE003)